src/lib/components/Meta.tsx

```diff
@@ -13,35 +15,46 @@ const Meta = () => {
       <meta name="apple-mobile-web-app-title" content={APP_NAME} />
       <meta name="format-detection" content="telephone=no" />
       <meta name="mobile-web-app-capable" content="yes" />
-      <meta name="theme-color" content="#FFFFFF" />
-      <title>{`${title} Explorer | Celatone`}</title>
+      <meta name="theme-color" content={CURR_THEME.colors.background.main} />
+      <title>{`${title} Explorer | ${CURR_THEME.branding.seo.title}`}</title>
       <meta
         name="description"
-        content="A smart contract powered explorer for the Cosmos."
+        content={CURR_THEME.branding.seo.description ?? FALLBACK_DESC}
```

Comment:
don't think we need all the FALLBACK variable since all variables in branding.seo are required